### ISSUE TITLE
Make App.all public

### ISSUE THREAD

**Issue by fluidsonic** [kiliankoe/Karte#30]
We'd like to do what #21 suggests directly in our app by simply checking which apps are installed.

Unfortunately the following won't work because `.all` isn't `public`:

```swift
let apps = App.all.filter(Karte.isInstalled)
if apps.count == 1 { … }
```

**Comment by fluidsonic** [comment]
Btw, writing your own `.all` isn't event needed. You can simply use the [`CaseIterable`](https://medium.com/@09mejohn/swift-4-2-enumerable-enum-748a2dca9e41) protocol on the enum.

**Comment by kiliankoe** [comment]
Ah, good point 👍 CaseIterable wasn't around yet when first writing this code.

**Comment by kiliankoe** [comment]
fixed with 8cded8997540e9607ef526dfaeb35055a96494e8
